### DISCUSSION TITLE
Can we open browser with remote_url ?

### DISCUSSION THREAD

**Discussion by dollyjung** [qahive/robotframework-puppeteer#158] [Q&A] [answered]
I am trying to use keywords open browser with remote_url, but i cannot see the option for this.
Please help suggest can we do this with robot framework - Puppeteer library.

**Comment by atthaboon** [comment, marked as answer]
Currently not support for remote_url

**Reply by dollyjung** [reply]
Hi Atthaboon, thank you. Could you please help suggest if we can do any workaround for the remote_url?

**Reply by atthaboon** [reply]
what's the objective for using the remote_url?
Is the web server behind the firewall?

**Reply by dollyjung** [reply]
the objective is to use remote_url to call Selenoid url.
